cime_config/config_grids.xml

```diff
@@ -4610,11 +4721,6 @@
       <map name="ROF2OCN_LIQ_RMAPNAME">cpl/cpl6/map_r05_to_oRRS15to5_nn.160203.nc</map>
     </gridmap>
 
-    <gridmap ocn_grid="oRRS18to6v3" rof_grid="r05">
-      <map name="ROF2OCN_ICE_RMAPNAME">cpl/cpl6/map_r05_to_oRRS18to6v3_smoothed.r50e100.211102.nc</map>
-      <map name="ROF2OCN_LIQ_RMAPNAME">cpl/cpl6/map_r05_to_oRRS18to6v3_smoothed.r50e100.211102.nc</map>
-    </gridmap>
-
```

Comment:
Same comment, seems unusual to change the RRS maps here

Reply:
Unless these are both just rebasing issues and will disappear?

Reply:
Oh, shoot!  Thanks for pointing that out. I think that was a mistake in a rebase that needs to be fixed.

Reply:
@vanroekel, this should be fixed now.  I believe I rebased onto a new `master` and @jonbob already had made changes to the grid files.  Then, when he added his changes after my rebase, it probably inadvertently undid the changes that had happened on `master`.

Thank you for noticing that!